pkg/diagnostics/component_monitoring.go

```diff
@@ -159,6 +169,23 @@ func (c *componentMetrics) PubsubIngressEvent(ctx context.Context, component, pr
 	}
 }
 
+// BulkPubsubEgressEvent records the metris for a pub/sub egress event.
+func (c *componentMetrics) BulkPubsubEgressEvent(ctx context.Context, component, topic string, success bool, elapsed float64) {
```

Comment:
@yaron2 @artursouza Do we need to have a separate operation count for num of times bulk publish is called and the status ? Does this seems ok?